e2e/fixtures/broken-links/src/redirects.ts

```diff
@@ -1,22 +1,23 @@
 import type { Middleware } from 'waku/config';
 
 const redirectsMiddleware: Middleware = () => async (ctx, next) => {
-  switch (ctx.req.url.pathname) {
-    case '/redirect':
-      ctx.res.status = 302;
-      ctx.res.headers = {
-        Location: '/exists',
-      };
-      break;
-    case '/broken-redirect':
-      ctx.res.status = 302;
-      ctx.res.headers = {
-        Location: '/broken',
-      };
-      break;
-    default:
-      return await next();
+  const redirects = {
+    '/redirect': '/exists',
+    '/RSC/R/redirect.txt': '/RSC/R/exists.txt',
+    '/broken-redirect': '/broken',
+    '/RSC/R/broken-redirect.txt': '/RSC/R/broken.txt',
+  };
```

Comment:
should this be declared outside the function scope? or does it not really matter?

Reply:
@dai-shi what do you think about exposing `encodeRscPath()` for this?